controller/instance_manager_controller.go

```diff
@@ -988,6 +999,15 @@ func (imc *InstanceManagerController) deleteInstanceManagerPDB(im *longhorn.Inst
 	return nil
 }
 
+func (imc *InstanceManagerController) syncOrphanedInstance(im *longhorn.InstanceManager) error {
+	if im.Spec.NodeID != imc.controllerID {
+		// The orphan owner node is no longer able to monitor instance status.
+		// Clear the orphan CRs, and they will be resync after the owner instance manager get back.
+		return imc.cleanupInstanceManagerOrphanInstances(im)
+	}
```

Comment:
Just a small note. We may need to be careful with this logic to avoid triggering actual data deletion